rpcd/playbooks/roles/elasticsearch/templates/legacy-mapping.sh.j2

```diff
@@ -0,0 +1,302 @@
+#!/bin/sh
+curl --retry 10 -XPUT 'http://{{ hostvars[inventory_hostname]['container_address'] }}:{{ elasticsearch_http_port }}/_template/legacy-liberty' -d '
+{
+  "template" : "*",
+  "order": 0,
+    "mappings" : {
+      "_default_" : {
+        "dynamic_templates" : [ {
```

Comment:
That's a straight dump from elasticsearch, and actually carried forward from liberty with a minor change.